values/jobs/autoconfig.gotmpl

```diff
@@ -0,0 +1,37 @@
+{{- $v := .Values }}
```

Comment:
could you please rename this file to `e2e.yaml`

Reply:
done

Reply:
done